build_scripts/build_wheel.sh

```diff
@@ -17,7 +17,8 @@
 #   For example
 #    ./build_scripts/build_wheel.sh . 0.1.0 10.2
 # NIGHTLY: if !=0, compiles a wheel for the nightly package (forces INCLUDE_LABS=1).
-#   THESEUS_VERSION is also ignored, since version is set to YYYY.MM.DD.
+#   THESEUS_VERSION and COMMIT are also ignored, since version is set to YYYY.MM.DD.
```

Comment:
This is just a comment, so I squeezed this in to avoid a separate commit for it.